src/BlockTypes/MiniCart.php

```diff
@@ -260,11 +260,10 @@ protected function append_script_and_deps_src( $script ) {
 			return;
 		}
 		$this->scripts_to_lazy_load[ $script->handle ] = array(
-			'src'          => $script->src,
-			'version'      => $script->ver,
-			'before'       => $wp_scripts->print_inline_script( $script->handle, 'before', false ),
-			'after'        => $wp_scripts->print_inline_script( $script->handle, 'after', false ),
-			'translations' => $wp_scripts->print_translations( $script->handle, false ),
```

Comment:
Looking closer, I notice that the `$wp_scripts->print_translations()` of enqueued scripts cause the JS errors; even we don't `echo` them, they are still printed.

Reply:
Not sure if related, but I remember being confused by something similar and it was because we hook into `pre_load_script_translations` to print the translations, which runs no matter if you `echo` them or not:
https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/2d0270cc23121f79a5d3f1759a7e7df43f1323a4/woocommerce-gutenberg-products-block.php#L239

Reply:
You're the life saver @Aljullu! Thanks so much for the suggestion!